charts/operator-wandb/templates/ingress.yaml

```diff
@@ -1,18 +1,5 @@
 {{- if .Values.ingress.issuer.create }}
-{{- if eq .Values.ingress.issuer.provider "google" }}
----
-apiVersion: networking.gke.io/v1
-kind: ManagedCertificate
-metadata:
-  name: {{ .Release.Name }}-cert
-  namespace: {{ .Release.Namespace }}
-  labels:
-    {{- include "wandb.commonLabels" . | nindent 4 }}
-spec:
-  domains:
-  - {{ replace "https://" "" (replace "http://" "" .Values.global.host) }}
-{{- else }}
----
+{{- if ne .Values.ingress.issuer.provider "google" }}
```

Comment:
`ne .Values.ingress.issuer.provider "google"` we can remove all these checks. If issuer is set to create, that use the logic, we don't need to check which type of provider is being set